osm2pgsql.cpp

```diff
@@ -45,6 +46,7 @@
 
 int main(int argc, char *argv[])
 {
+    std::setlocale(LC_NUMERIC, "C");
```

Comment:
This may help to avoid problem on Linux with changed locale (did not help on Windows, however)